### ISSUE TITLE
can't deploying app

### ISSUE THREAD

**Issue by DayatXD** [ayastreb/bandwidth-hero-proxy#30]
its say in the log that:
!     Push rejected, failed to compile Node.js app.
 !     Push failed

**Comment by andydev404** [comment]
You're right, to fix that you need to clone the project and update the sharp version to the ^0.28.1 after that you can manually deploy the project to heroku, and should work.

**Comment by daminduLiyanage** [comment]
Remember to update the node engine version number. It works for ^15.4.0 if that helps.
